lib/pr_checker.js

```diff
@@ -185,7 +185,7 @@ class PRChecker {
       const dateStr = new Date(pr.createdAt).toDateString();
       const type = wait.isWeekend ? 'weekend' : 'weekday';
       cli.info(`This PR was created on ${dateStr} (${type} in UTC)`);
-      cli.warn(`${wait.timeLeft} hours left to land`);
+      cli.warn(`Wait at least ${wait.timeLeft} hours before landing`);
```

Comment:
Optional nit: Maybe to avoid more confusion make it "Wait at least N **more** hours before landing"? So that someone doesn't see that it says to wait 4 hours before landing, look at their PR and think "it's been open for two days, I can land it now because I waited 4 hours!"

Reply:
4530c4692e0b613d76f4db014549498afa520f79